### PR TITLE
Apply ufmt linter to all py files under test/onnx

### PR DESCRIPTION
**Pull request by huydhn**
Same as https://github.com/pytorch/pytorch/pull/81285 but for `test/onnx`. The merge conflicts in `linrunner.toml` is expected. I will resolve them depending on the merge order of the PR

### CODE REVIEW

**Comment by justinchuby** [comment]
onnx tests are previously formatted with isort. Are we moving toward usort? Is there a way to make them compatible?

**Comment by huydhn** [comment]
> onnx tests are previously formatted with isort. Are we moving toward usort? Is there a way to make them compatible?

I think you are referring to this [commit](https://github.com/pytorch/pytorch/commit/5dd1c677768a94ee4c014438369c8a9f832a7e40) in which these files were formatted by isort and black. There are several gaps here that I try to address:

* There is currently no [linter adapter](https://github.com/pytorch/pytorch/tree/master/tools/linter/adapters) defined for both isort and usort. So none of them will be enforced automatically
* isort/usort and black should be wrapped into just one linter, which is what I have recently introduced in the new [ufmt linter](https://github.com/pytorch/pytorch/blob/master/tools/linter/adapters/ufmt_linter.py). [ufmt](https://ufmt.omnilib.dev/en/stable/) is a nice little tool that wraps around black and usort (instead of isort). IMO, this should make things easier to maintain in the long run. This is also the main reason why I mention usort here
* To answer your question, isort and usort outputs are almost identical. But usort is built as a replacement of isort to only sort import when it's considered safe to do so.. Thus it sounds like a better choice than isort

As for their style differences as shown in this PR, I guess they are due to the mindset of usort developers as quoted from their wiki:

> As for style preferences, or the desire to "group" imports based on "family" of modules, it's our position that this is both difficult to correctly catalog and implement in software, and also reduces overall ability to maintain projects or transfer knowledge between teams as it depends on humans to understand and keep this per-project context in memory, as opposed to relying on canonical heuristics. By sorting into the fewest number of categories as possible, and maintaining a strict lexicographical ordering within each category, new and veteran engineers alike can more easily find and update imports with the least amount of consideration and mental context switching while working on their code.

So I see imports like:
```
import sys

import onnx
import pytorch_test_common

import caffe2.python.onnx.backend as c2
import torch
import torch.jit
from torch.autograd import Variable
```

,which isort is happy with, become:

```
import sys

import caffe2.python.onnx.backend as c2

import onnx
import pytorch_test_common
import torch
import torch.jit
from torch.autograd import Variable
```

**Comment by justinchuby** [comment]
I personally like the way isort groups things. It is also how pep8 suggested imports to be sorted (https://peps.python.org/pep-0008/#imports). That said it is great to have consistent and enforced formatting. Thanks for the pr!

File: test/onnx/test_models_onnxruntime.py [hunk 1]
```diff
@@ -8,9 +8,13 @@
 import onnx_test_common
 import parameterized
 import PIL
+
+import torch
 import torchvision
 from pytorch_test_common import skipIfUnsupportedMinOpsetVersion, skipScriptTest
 from test_models import TestModels
+from torch import nn
+from torch.testing._internal import common_utils
```
Comment (justinchuby) [inline test/onnx/test_models_onnxruntime.py:17]: Although breaking things up like this is still... hmm its ok I guess

**Comment by huydhn** [comment]
Ping @BowenBao

**Review by BowenBao — APPROVED** [review]
🚀

**Review by kit1980 — APPROVED** [review]

**Pull request closed without merging**